src/Microsoft.Sbom.Api/Executors/ComponentDetectionBaseWalker.cs

```diff
@@ -144,21 +144,20 @@ async Task Scan(string path)
                     licenseInformationRetrieved = true;
 
                     List<string> apiResponses;
-                    var licenseInformationFetcher2 = licenseInformationFetcher as ILicenseInformationFetcher2;
-                    var licenseInformationTimeoutInSecondsConfigSetting = GetLicenseInformationTimeoutInSecondsSetting(configuration);
+                    var licenseInformationTimeoutInSecondsConfigSetting = configuration.LicenseInformationTimeoutInSeconds;
 
-                    if (licenseInformationFetcher2 is null && (bool)!licenseInformationTimeoutInSecondsConfigSetting?.IsDefaultSource)
+                    if (licenseInformationFetcher is null && (bool)!licenseInformationTimeoutInSecondsConfigSetting?.IsDefaultSource)
                     {
                         log.Warning("Timeout value is specified, but ILicenseInformationFetcher2 is not implemented for the licenseInformationFetcher");
```

Comment:
I think this warning can be removed now

Reply:
Thanks--I missed a bunch of changes. Should be fixed in the next commit